llvm/utils/UpdateTestChecks/common.py

```diff
@@ -1079,6 +1079,7 @@ def get_value_use(self, var, match, var_prefix=None):
     NamelessValue(r"META", "!", r"", r"![0-9]+", r"(?:distinct |)!.*"),
     NamelessValue(r"ACC_GRP", "!", r"!llvm.access.group ", r"![0-9]+", None),
     NamelessValue(r"META", "!", r"![a-z.]+ ", r"![0-9]+", None),
+    NamelessValue(r"META", "!", r".", r"![0-9]+", None),
```

Comment:
It would be good to make the prefix expression a little more specific if possible. It can probably be replaced with `r" "`, right? Or `r"[( ]"`?

And given that, can you remove the `NamelessValue` with the "metadata " prefix? It seems that this is now redundant.

Reply:
Interesting point. The tool really is meant to look at tooling output, not human-written IR, so you can argue both ways. I don't feel very strongly about it -- I think either way is fine.